client/src/components/modal-forms/NewParticipantForm.js

```diff
@@ -76,11 +77,11 @@ export const NewParticipantForm = ({ initialValues, validationSchema, onSubmit,
           />
           <Field
             name="site"
-            component={RenderSelectField}
+            component={RenderAutocomplete}
```

Comment:
So this component can be used anywhere that we currently use SelectField, cool.

Reply:
That's the hopeful idea! I haven't tested elsewhere but it's at least a drop-in for this use-case